packages/nimble-components/src/theme-provider/design-token-names.ts

```diff
@@ -239,7 +239,25 @@ export const tokenNames: { readonly [key in TokenName]: string } = {
     mentionFontFamily: 'mention-font-family',
     mentionFontSize: 'mention-font-size',
     mentionFontWeight: 'mention-font-weight',
-    mentionFontLineHeight: 'mention-font-line-height'
+    mentionFontLineHeight: 'mention-font-line-height',
```

Comment:
Let's try to get consensus on naming. [These are the guidelines](https://github.com/ni/nimble/blob/main/packages/nimble-components/CONTRIBUTING.md#naming), with the expected format being `[element]-[part]-[interaction_states]-[remaining_states]-[token_type]`. And here's a screenshot showing some of these states (not using these tokens yet).
![image](https://github.com/user-attachments/assets/b5d71dd6-91df-448f-a672-87a40a9c0bc2)
 
I'd propose these values for "element":
1. `calendar` - everything except the cards
2. `calendar-event` - a card in any of its states
3. `grab-handle` - the circle that you can click on to resize a calendar event (I suggest making this a separate "element" since I'm guessing it'd be the same styling if we allowed resizing other components).
4. I'm not sure what "calendar-resource-selection" is so not sure if it's an element or a part. Could you clarify with a screenshot?

Then `calendar-event` could have "remaining_states" like "static", "dynamic", "transient". Maybe "blocked" too? Does that behave like an event (a rounded rectangle within a time slot) or is it a part of a calendar? (I'll admit I'm on the fence about putting this at the end. It feels clunky but seems to fit our guidance best)

We should add the "part" for the "hover" tokens (background, I assume?) and grab-handle/resize-ring.

For example, this would yield:
- `calendar-event-background-static-color`
- `calendar-event-background-dynamic-color`
- ...
- `calendar-event-border-static-color`
- ...
- `calendar-event-static-font-color`
- ...
- `calendar-event-background-hover-dynamic-color`
- ...
- `grab-handle-background-color`

Thoughts @rajsite?

Reply:
Oh, missed this before my other comments, but yea this is more inline with what I would expect for the naming 👍

Reply:
> Then calendar-event could have "remaining_states" like "static", "dynamic", "transient". Maybe "blocked" too? Does that behave like an event (a rounded rectangle within a time slot) or is it a part of a calendar?

These states are just a different appearance of the events as below:

Static Event:

![image](https://github.com/user-attachments/assets/402a1218-4cea-4275-9512-9a0f33b94392)

These are static events that represent the actual/defined data (In our case test plans data scheduled against systems )

Dynamic Event:

![image](https://github.com/user-attachments/assets/1c003eaf-672e-4bf6-b4f3-4e7f814c9f95)

This represents that the event can be dynamically created/resize ( In our case its the event created by the user to select the timeline for the current test plan to be scheduled)

Transient Event:

![image](https://github.com/user-attachments/assets/e5e2f235-157c-449c-a619-bfcc95a12731)

These don't have specific state, They will just act as a reflection of the Dynamic events.


I have updated all the tokens names as per guidance, Please let me know if anything need to be updated.

Reply:
Generally looks good to me. I left a couple more comments on the threads Milan had opened. And one last point is whether the grab handle should be `calendar-grab-handle` or just `grab-handle`. I originally proposed the latter but on second thought I think I prefer to start with it scoped to calendar since we have fairly low confidence that the design will apply generally.